lib/graphql/schema.ts

```diff
@@ -530,14 +542,6 @@ export enum DocumentType {
   Invoice = 'INVOICE'
 }
 
-export type ElsterGroup = {
```

Comment:
❓ Same question as before

Reply:
all of this is automatically generated from backend. Because backend has `elsterCode` in request/response, so it's generated like this.